### PR TITLE
Add manifest for pass-winmenu-nogpg

### PR DESCRIPTION
**Pull request by pakdev**
This version of pass-winmenu is for people that already have gpg installed.

### CODE REVIEW

File: bucket/pass-winmenu-nogpg.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "description": "A simple, easy-to-use password manager for Windows that follows the philosophy of (and is compatible with) the Linux password manager [pass](https://www.passwordstore.org/).",
+    "license": "MIT",
+    "url": "https://github.com/Baggykiin/pass-winmenu/releases/download/v1.9/pass-winmenu-nogpg.zip",
+    "hash": "466436be3dfa5e250b2739ec5e3358d351a7b7c5e2f4564ce335592b36abb670",
+    "bin": "pass-winmenu\\pass-winmenu.exe",
+    "checkver": {
+        "github": "https://github.com/Baggykiin/pass-winmenu/"
+    },
+    "autoupdate": {
+        "url": "https://github.com/Baggykiin/pass-winmenu/releases/download/v$version"
```
Comment (Ash258) [inline bucket/pass-winmenu-nogpg.json]: Wrong url
```suggestion
        "url": "https://github.com/Baggykiin/pass-winmenu/releases/download/v$version/pass-winmenu-nogpg.zip",
```

File: bucket/pass-winmenu-nogpg.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1.9",
+    "homepage": "https://github.com/Baggykiin/pass-winmenu",
+    "description": "A simple, easy-to-use password manager for Windows that follows the philosophy of (and is compatible with) the Linux password manager [pass](https://www.passwordstore.org/).",
+    "license": "MIT",
+    "url": "https://github.com/Baggykiin/pass-winmenu/releases/download/v1.9/pass-winmenu-nogpg.zip",
+    "hash": "466436be3dfa5e250b2739ec5e3358d351a7b7c5e2f4564ce335592b36abb670",
+    "bin": "pass-winmenu\\pass-winmenu.exe",
```
Comment (Ash258) [inline bucket/pass-winmenu-nogpg.json]:
```suggestion
    "extract_dir": "pass-winmenu",
    "bin": "pass-winmenu.exe",
```

File: bucket/pass-winmenu-nogpg.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "checkver": {
```
Comment (Ash258) [inline bucket/pass-winmenu-nogpg.json]:
```suggestion
    "checkver": "github",
```

File: bucket/pass-winmenu-nogpg.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1.9",
+    "homepage": "https://github.com/Baggykiin/pass-winmenu",
+    "description": "A simple, easy-to-use password manager for Windows that follows the philosophy of (and is compatible with) the Linux password manager [pass](https://www.passwordstore.org/).",
```
Comment (Ash258) [inline bucket/pass-winmenu-nogpg.json]: Do not use markdown syntax in notes. It's not for nothing.

```suggestion
    "description": "A simple, easy-to-use password manager for Windows that follows the philosophy of (and is compatible with) the Linux password manager pass.",

```

**Comment by Ash258** [comment]
It should be only command line, so move this PR into main bucket. Also add depends for gpg, since it's required.

File: bucket/pass-winmenu-nogpg.json (hunk 2 above)
Comment (pakdev) [reply bucket/pass-winmenu-nogpg.json]: I assume bin becomes `pass-winmenu.exe` when I add `extract_dir`, right?

File: bucket/pass-winmenu-nogpg.json (hunk 4 above)
Comment (Ash258) [reply bucket/pass-winmenu-nogpg.json]: There is no need for url at all. If you want it, just add it to normal brackets surrounded with `'` to keep clickable url in terminal.

File: bucket/pass-winmenu-nogpg.json (hunk 2 above)
Comment (Ash258) [reply bucket/pass-winmenu-nogpg.json]: Yes. As in suggestion:

1 line removed, 2 added,

S

Comment (pakdev) [reply bucket/pass-winmenu-nogpg.json]: Doh; my bad

**Comment by pakdev** [comment]
While pass is CLI, pass-winmenu is GUI. Also, my intention was to create both pass-winmenu and pass-winmenu-nogpg. For the time being, though, I only need pass-winmenu-nogpg.

**Comment by Ash258** [comment]
Ok. 
Then only add suggest for gnupg, gnupg1 and gpg, persist for `pass-winmenu.yaml` and shortcuts to enable running from starts menu, when it's GUI application.

```json
"suggest": {
    "GPG: [
         "gnupg",
         "gnupg1",
         "gpg",
    ]
},
"persist": "pass-winmenu.yaml",
"shortcut": [
    [
        "pass-winmenu.exe",
        "Pass winmenu"
    ]
]
```

**Comment by pakdev** [comment]
I presume the GPG suggests should come from scoop itself, of which, I only see these:
image

**Comment by pakdev** [comment]
Is there anything else you need from me to get this pulled?

**Pull request merged**